runtime/j9vm/java11vmi.c

```diff
@@ -1890,3 +1890,15 @@ JVM_IsUseContainerSupport(void)
 
 	return result;
 }
+
+#if JAVA_SPEC_VERSION >= 17
+/**
+ * @brief Get the default system properties.
+ *
+ * @return an Object array of system property key value pairs
+ */
+jobjectArray JVM_GetProperties(JNIEnv *env)
+{
+	return NULL;
```

Comment:
The extensions code is not expecting `null`.

Reply:
Until more changes are made, it won't be called.

Reply:
Won't it be used if https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/952 is merged?
If not, this should just assert (like `JVM_GetSimpleBinaryName()`).

Reply:
added the assert